src/StarMatch.cc

```diff
@@ -65,10 +64,10 @@ std::ostream &operator<<(std::ostream &stream, const StarMatchList &starMatchLis
     return stream;
 }
 
-static std::unique_ptr<double[]> chi2_array(const StarMatchList &starMatchList,
+static std::vector<double> chi2_array(const StarMatchList &starMatchList,
                                             const AstrometryTransform &transform) {
     unsigned s = starMatchList.size();
-    auto res = std::unique_ptr<double[]>(new double[s]);
+    std::vector<double> res(s,0.);
```

Comment:
While we're at it, lets replace `res`->`result` and `s`->`size`.

Reply:
Done.
    `std::vector<double> result` also does not need  to be default intialized